### PR TITLE
Update late applicants page based on policy feedback

### PR DESCRIPTION
**Pull request by MylesJarvis**
We've received some feedback on this page (apply for teacher training this September) from the policy SME. Updating content to reflect some of this.

### CODE REVIEW

File: app/views/content/start-teacher-training-this-september.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ Our teacher training advisers can help you write a strong application quickly.
 They will help you to:
```
Comment (MylesJarvis) [inline app/views/content/start-teacher-training-this-september.md:30]: Check line 28 for potential duplicated content - this relates to my Q in the initial PR comments.

File: app/views/content/start-teacher-training-this-september.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ Our teacher training advisers can help you write a strong application quickly.
...
 
 * create a personal statement that shows off your strengths
-* choose the best people to be your references
-* explain any gaps in your work history
+* submit a quality application, including choosing the right references
```
Comment (peteryates) [inline app/views/content/start-teacher-training-this-september.md:33]: Not sure about the word 'quality' on its own here

**Review by peteryates — APPROVED** [review]
Looks fine to me overall, one question on word choice but it's good to go as-is if needed

File: app/views/content/start-teacher-training-this-september.md (hunk 2 above)
Comment (Tomcowan1) [reply app/views/content/start-teacher-training-this-september.md:33]: I'd just take this bullet out - references are important on their own, but if the ITT would rather not draw this out then that's fine. I don't think "quality application" adds anything given the other content.

**Review by Tomcowan1 — APPROVED** [review]

**Pull request merged**